keras_core/layers/preprocessing/discretization_test.py

```diff
@@ -41,36 +40,34 @@ def test_correctness(self):
             bin_boundaries=[0.0, 0.5, 1.0], output_mode="int"
         )
         output = layer(np.array([[-1.0, 0.0, 0.1, 0.8, 1.2]]))
-        self.assertTrue(backend.is_tensor(output))
```

Comment:
The is_tensor checks should be kept. The output of the layer should always be a backend native tensor.

Reply:
done